### ISSUE TITLE
Can't release at arm32.

### ISSUE THREAD

**Issue by Git-David-Dai** [Shopify/mobile-buy-sdk-ios#284]
```
- (NSOperation *)getProductTagsPage:(NSUInteger)page completion:(BUYDataTagsListBlock)block
{
	NSURL *url  = [self urlForProductTagsWithParameters:@{
														  @"limit" : @(self.pageSize),
														  @"page"  : @(page),
														  }];
	
	return [self getRequestForURL:url completionHandler:^(NSDictionary *json, NSHTTPURLResponse *response, NSError *error) {
		
		NSArray *tags = nil;
		if (json && !error) {
			tags = [json[@"tags"] valueForKey:@"title"];
		}
		block(tags, [self hasReachedEndOfPage:tags], page, error);
	}];
}

```
It seem that,that block parameter have some problem.
also i found that demo can't run on iphone it will  be crash for the reason:
dyld: Library not loaded: @rpath/Buy.framework/Buy
  Referenced from: /var/containers/Bundle/Application/90CAD955-65C0-48E5-9500-D06DCF8CEA13/Mobile Buy SDK Advanced Sample.app/Mobile Buy SDK Advanced Sample
  Reason: image not found

**Comment by gabrieloc** [comment]
It looks like this was addressed by https://github.com/Shopify/mobile-buy-sdk-ios/pull/280. Can you try again?

**Comment by bgulanowski** [comment]
Fixed by #280
